app/src/main/java/com/github/orkest/View/auth/SignUpForm.kt

```diff
@@ -11,51 +13,72 @@ import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
 import com.github.orkest.Model.Providers
+import com.github.orkest.View.MainActivity
 import com.github.orkest.View.theme.White
 import com.github.orkest.ViewModel.auth.AuthViewModel
 
 
-// Needs to be initialized in the main activity and called as a callback function
-// after the google authentication sign up returns
+// Needs to be initialized in the auth activity and called as a callback function
+// after the google authentication sign in returns
 @SuppressLint("UnusedMaterialScaffoldPaddingParameter")
 @Composable
 fun SignUpForm(navController: NavController, viewModel: AuthViewModel) {
+
+    // Value storing the context, used later to launch the Main Activity
+    val context = LocalContext.current
+
+    // Value used for the dropDown menu selecting the providers
+    val expanded = remember { mutableStateOf(false) }
+
+    // Value used to display an error message if the username already exists
+    val userExists = remember {
+        mutableStateOf(false)
+    }
+
     Scaffold(
+        //Creates the top bar
         topBar = {
             TopAppBar(
                 title = { Text(text = "Create Your Profile") },
             )},
+
         content = {
             Column(
                 modifier = Modifier.padding(20.dp),
                 verticalArrangement = Arrangement.Center,
                 horizontalAlignment = Alignment.CenterHorizontally
             ) {
-
-                val expanded = remember { mutableStateOf(false) }
-
-
+                //Title of the page
                 Text(
                     text = "Create Your Profile",
                     style = TextStyle(fontSize = 40.sp, fontFamily = FontFamily.Cursive)
                 )
 
                 Spacer(modifier = Modifier.height(15.dp))
```

Comment:
You repeat this same line a few times in the code. Unless you intend on modifying the space between elements frequently I think you could store this into a variable.